### DISCUSSION TITLE
What about keybindings manual?

### DISCUSSION THREAD

**Discussion by AndriiHaranin** [crivotz/nv-ide#41] [Q&A]
I did not find not a word about how to understand the keybindings for this magnificent config :(

**Comment by crivotz** [comment]
Thanks 😄 
Rightly you are not the first to ask me, mainly mappings are in the file https://github.com/crivotz/nv-ide/blob/master/lua/config/mappings.lua
Instead, the plugin `which-key` it helps more during the use of the configuration itself

However I will follow your tip and I will add some info to the README (issue #42 )
